opencodelists/db_utils.py

```diff
@@ -1,8 +1,85 @@
 from django.db import DEFAULT_DB_ALIAS, connections
 
+from codelists.coding_systems import CODING_SYSTEMS
+
 
 def query(sql, params=None, database=None):
     database = database or DEFAULT_DB_ALIAS
     with connections[database].cursor() as c:
         c.execute(sql, params)
         return c.fetchall()
+
+
+class CodingSystemVersionRouter:
+    """
+    A router to ensure coding system models always use a named version database.
+
+    For the most part, coding system databases are only used for reads, and will
+    accessing the database with a `using` argument which will bypass this router,
+    e.g. `Concept.objects.using("snomedct_v1_20221001").filter(...)`.
```

Comment:
I don't understand this paragraph - could you try rewording?  It's partly because there's a missing word or something ("and will accessing the database").

But also, when reading this, I'd find it really useful to have an explanation of the rationale of having a different database for each version, and how the components of `<coding_system>_<release_name>_<valid_from date>` are meaningful, where.

I've not been following closely, so perhaps this explanation is elsewhere in the code -- if so, linking to that might be useful? I can imagine a developer getting an error from this router, not really understanding it, and coming to this part of the codebase first.